src/simulation/m_global_parameters.fpp

```diff
@@ -1021,12 +1066,16 @@ contains
         ! the next one
         if (viscous) then
             buff_size = 2*weno_polyn + 2
-!        else if (hypoelasticity) then !TODO: check if necessary
-!            buff_size = 2*weno_polyn + 2
         else
             buff_size = weno_polyn + 2
         end if
 
+        if (elasticity) then
+            fd_order = 4
```

Comment:
fd_order is a case file parameter. I don't think this should be set to a magic number in the code. I think it should remain in the case file, and a check should be added to m_checker_common if fd_order = 4 is in fact necessary when using elasticity.